_Mod/Modders Resource/YayoFactory/Defs/ThingDefs/SynthYayo.xml

```diff
@@ -0,0 +1,201 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<SynthYayo>
+
+    <ThingDef ParentName="MakeableDrugBase">
```

Comment:
"MakeableDrugBase" has not been defined in this mod.  Be sure to define all abstracts you use, they are scoped to the mod that defines them.